pkg/cpio/fs_plan9.go

```diff
@@ -100,6 +104,22 @@ func CreateFileInRoot(f Record, rootDir string, forcePriv bool) error {
 			return err
 		}
 
+	case os.ModeSymlink:
+		b, err := ioutil.ReadAll(uio.Reader(f))
+		if err != nil {
+			return err
+		}
+		if err := ioutil.WriteFile(f.Name, []byte{}, 0444); err != nil {
+			return err
+		}
+		content := string(b)
+		if !filepath.IsAbs(content) {
+			content = filepath.Join(dir, content)
+		}
+		if err := syscall.Bind(f.Name, content, syscall.MREPL); err != nil {
```

Comment:
Would be worth adding a plan9-specific flag at some point to decide whether to bind or duplicate the file.  This works well for now though.

Reply:
I'ld like to wait until we need it.